config/routes.rb

```diff
@@ -13,4 +13,9 @@
   put '/api/hideouts/api/users', to: 'hideout#add'
   delete '/api/hideouts/api/users', to: 'hideout#leave'
   delete '/api/hideouts/api/destroy', to: 'hideout#destroy'
+
+  post '/api/chores', to: 'chores#create'
+  put '/api/chores/:id', to: 'chores#update'
+  delete '/api/chores/:id', to: 'chores#destroy'
```

Comment:
Can we also add a `GET /api/chores/:id` route? It would be very easy to code, just create a `show` method that legit just looks up the chore by id and returns the object calling `chore.as_json` when returning. Add a comment that says we assume chore and user exists as always.